SwiftUIProject/SwiftUIProject/ContentView.swift

```diff
@@ -6,19 +6,151 @@
 
 import SwiftUI
 
+/// Settings screen
 struct ContentView: View {
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let zeroValue: Double = 0
+        static let twentyTwoValue: CGFloat = 22
+        static let thirtyFiveValue: CGFloat = 35
+        static let seventyValue: CGFloat = 70
+        static let twentyValue: CGFloat = 20
+        static let backwardImageName = "backward.fill"
+        static let pauseImageName = "pause.fill"
+        static let playImageName = "play.fill"
+        static let forwardImageName = "forward.fill"
+        static let squareArrowImageName = "square.and.arrow.down"
+        static let saveInLibraryText = " save in library"
+        static let greatText = "Great!"
+        static let arrowshapeImageName = "arrowshape.turn.up.right"
+        static let wantShareText = "You want to share?"
+        static let yesText = "Yes"
+        static let noText = "No"
+    }
+    
+    // MARK: - Public Properties
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+            makeAlbumImage
+            makeSliderAndTime
+            makeSongName
+            HStack {
+                makeBackwardButton
+                makePauseButton
+                makePlayButton
+                makeForwardButton
+            }
+            HStack {
+                makeSaveButton
+                makeShareButton
+            }
+        }
+    }
+    
+    // MARK: - Private Properties
+    
+    @ObservedObject var viewModel = PlayerView()
+    @State private var isSaveTrackActionSheetShown = false
+    @State private var isSharedAlertShown = false
+    
+    private var makeAlbumImage: some View {
```

Comment:
```suggestion
    private var albumImageView: some View {
```
где возвращаем some View на конце пишем View